### ISSUE TITLE
[Important]: Unmaintained project

### ISSUE THREAD

**Issue by leipert** [leipert/language-pfm#31]
Dear users of language-pfm (there do not seem to be too many of you!),

I am not using Atom anymore and therefore I am not interested in maintaining this Plugin anymore. Sorry for the long silence and not answering pull requests/issues, but I simply do not have the time anymore. If someone wants to step up as a maintainer, I am happy to talk to you!

Best,

@leipert

**Comment by robinrosenstock** [comment]
@leipert  out of curiosity: what do you use instead of atom?

**Comment by leipert** [comment]
@geniusupgrader 
I am using WebStorm during my day job, currently looking at vscode, but I think I'll use vim or sublime as my go-to text editors, because I am really annoyed by the slowness of atom/vscode.

For note-taking I used boostnote https://boostnote.io/, but I am considering using Bear http://www.bear-writer.com/.

When it comes to editing markdown, I mainly use it online anyways and just once a year with pandoc. So it really makes no sense for me to invest time in those plugins.

It's a bit sad, but interests shift over time, a few years I always was like: I will always update my OSS projects and keep them updated, but it is really hard once you start working full-time and/or develop interests/hobbies that are not connected to software.

**Comment by robinrosenstock** [comment]
@leipert thanks for answering. Yeah.. maybe you're right, interests shift over time..

**Comment by scotartt** [comment]
I am using sublime text (in the main) and Pandoc markdown to write my PhD thesis, the Pandoc support is spotty at best. I'd volunteer to maintain the package for Atom, but I'm just about to submit, and looking at Atom, probably as a way to procrastinate. The bibtex referencing thing is awesome, I came here to raise an issue I've found with the PFM syntax, and then saw this entry. Thanks for your effort! Maybe once I've finished my PhD I'll for this package and see what I can do to improve it.
